gulp/util/webpack-config.js

```diff
@@ -56,7 +56,6 @@ const EXTERNALS = {
     "jquery": "$",
     "moment": "moment",
     "react": "React",
-    "react-addons-css-transition-group": "React.addons.CSSTransitionGroup",
```

Comment:
this deletion is missing a corresponding addition. i suspect you'll need to add an entry for `react-transition-group`, like you've done everywhere else, so the library is excluded from `core.bundle.js`.